kevlar/call.py

```diff
@@ -161,79 +170,128 @@ def call_snv(target, query, offset, length, ksize):
         window = q[minpos:maxpos]
         refrwindow = t[minpos:maxpos]
 
-        numoverlappingkmers = len(window) - ksize + 1
-        kmers = [window[i:i+ksize] for i in range(numoverlappingkmers)]
+        # numoverlappingkmers = len(window) - ksize + 1
+        # kmers = [window[i:i+ksize] for i in range(numoverlappingkmers)]
```

Comment:
No longer used. Leaving it here for now in case it is needed in the near future.